worker/src/routes/docs.ts

```diff
@@ -50,6 +54,22 @@ export function resolveURL(path: string, ctx: Context) {
 
   log.debug('[Docs/resolve] resolved path: ', resolvedPath);
 
+  if (branch) {
+    const req = await fetch(`https://api.github.com/repos/${PREVIEW_REPO_OWNER}/${PREVIEW_REPO_NAME}/commits/${ref}`, {
+      headers: {
+        // TODO replace user-agent and add Auth header to increase rate limits
```

Comment:
I think storing in the worker secrets is the way to go for now.. since the repo will always be public I think a public read-only PAT would be fine. I just made one and added it as `GITHUB_PAT`